project/core/settings.py

```diff
@@ -37,6 +37,8 @@
 
 CSRF_TRUSTED_ORIGINS = env.list("DJANGO_CSRF_TRUSTED_ORIGINS", default=[])
 
+INTERNAL_IPS = env.list("INTERNAL_IPS", default=[])
```

Comment:
Do developers need to set this environment variable when working locally? If so, we should add a note to our contributing guide explaining the variable and the expected value. 

Alternatively, we might set the default to something other than `[]`

Reply:
I was not using the IP in my local.